umm-spec-lib/test/cmr/umm_spec/test/migration/version/variable.clj

```diff
@@ -181,3 +229,35 @@
                                                             {:GroupPath "/MODIS_Grid_Daily_1km_LST/Data_Fields"
                                                              :IndexRanges {:LatRange [-45 45]
                                                                            :LonRange [90 180]}})))))
+
+(deftest migrate-13->14
```

Comment:
nit: I prefer the variable map be included in the test over a `def`. This way I can easily see what is being migrated. Also, we don't need to include irrelevant fields in the variable map.

Reply:
I'm only using def because all the previous versions are using def in the test.  I don't mind using a more realistic variable for the tests.